.github/workflows/update-changelog.yml

```diff
@@ -11,6 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
```

Comment:
I saw your work on the `main-test` branch, but I think a lot of information/insight you have determined could be shared/incorporate here (here is a question we should answer with a comment IMO)

* Why do we want to _not_ persist credentials? is it because otherwise it might accidentally result in the Github action being used for pushes/etc. instead of our specified https url credentials for the Angular robot?